README.md

```diff
@@ -1,7 +1,57 @@
 
-# income-tax-state-benefits-frontend
+# income-tax-state-benefits
+
+This is where we make API calls from users viewing and making changes to the State Benefits section of their income tax return.
+
+## Running the service locally
+
+You will need to have the following:
+- Installed/configured [service-manager](https://github.com/hmrc/service-manager)
+
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_STATE_BENEFITS_FRONTEND
+
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+To run the service locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+    sm --stop INCOME_TAX_STATE_BENEFITS_FRONTEND
+    ./run.sh **OR** sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes run
+
+This service runs on port: `localhost:9376`
```

Comment:
Please move this after the line where Income Tax State Benefits FE service is started to make it clear that the port is for that service, not for the other ones

Reply:
Moved